pkg/apiserver/api_installer.go

```diff
@@ -379,18 +384,22 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				Filter(m).
 				Doc("partially update the specified "+kind).
 				Consumes(string(api.JSONPatchType), string(api.MergePatchType), string(api.StrategicMergePatchType)).
-				Operation("patch" + kind).
+				Operation("patch"+kind).
 				Produces(append(storageMeta.ProducesMIMETypes(action.Verb), "application/json")...).
-				Reads(versionedObject)
+				Returns(http.StatusOK, "OK", versionedObject).
+				Reads(versionedObject).
+				Writes(versionedObject)
 			addParams(route, action.Params)
 			ws.Route(route)
 		case "POST": // Create a resource.
 			route := ws.POST(action.Path).To(CreateResource(creater, reqScope, a.group.Typer, admit)).
 				Filter(m).
-				Doc("create a " + kind).
-				Operation("create" + kind).
+				Doc("create a "+kind).
+				Operation("create"+kind).
 				Produces(append(storageMeta.ProducesMIMETypes(action.Verb), "application/json")...).
-				Reads(versionedObject)
+				Returns(http.StatusOK, "OK", versionedObject).
+				Reads(versionedObject).
+				Writes(versionedObject)
 			addParams(route, action.Params)
 			ws.Route(route)
 		case "DELETE": // Delete a resource.
```

Comment:
This shows up as `*v1beta1.Status` in the swagger output. You need to deref the pointer.